packages/launcher/lib/linux/index.ts

```diff
@@ -17,7 +17,7 @@ function getLinuxBrowser (
     }
 
     log(
-      'Could not extract version from %s using regex %s',
+      'Could not extract version from "%s" using regex %s',
       stdout,
       versionRegex,
```

Comment:
I'm more partial to using the object notation for things like this, since it will include the variable names and ensure strings are quoted correctly:

```suggestion
      'Could not extract version from stdout %o', { stdout, versionRegex },
```

Reply:
Updated with object